swaglyrics/cli.py

```diff
@@ -105,10 +105,11 @@ def lyrics(song: str, artist: str, make_issue: bool = True) -> str:
 	lyrics = get_lyrics(song, artist)
 	if not lyrics:
 		lyrics = f"Couldn't get lyrics for {song} by {artist}.\n"
-		# Log song and artist for which lyrics couldn't be obtained
+		# log song and artist for which lyrics couldn't be obtained
 		with open(unsupported_txt, 'a', encoding='utf-8') as f:
 			f.write(f'{song} by {artist} \n')
-		if make_issue:
+		if make_issue and re.search(aln, song + artist):
```

Comment:
Right, so I was thinking that if the song artist name is trivial then we should say something to the user that lyrics may not be available on genius